control_panel_api/services.py

```diff
@@ -138,8 +139,10 @@ def create_bucket(bucket_name):
 
 def create_bucket_policies(bucket_name):
     """Create readwrite and readonly policies for s3 bucket"""
-    aws.create_policy(_policy_name(bucket_name, readwrite=True), get_policy_document(bucket_name, True))
-    aws.create_policy(_policy_name(bucket_name, readwrite=False), get_policy_document(bucket_name, False))
+    aws.create_policy(_policy_name(bucket_name, readwrite=True),
+                      get_policy_document(bucket_name, True))
+    aws.create_policy(_policy_name(bucket_name, readwrite=False),
+                      get_policy_document(bucket_name, False))
```

Comment:
Please avoid arbitrary indentation levels. Break the line after the open parenthesis and indent 4 spaces.
```python
aws.create_policy(
    _policy_name(...
```

Reply:
This follows pep8, and is auto formatted, see first example

https://www.python.org/dev/peps/pep-0008/#indentation

Do we have a custom code style to follow I can reference?

Reply:
This is what PEP8 says about indentation: https://www.python.org/dev/peps/pep-0008/#indentation

I tend to write stuff like this as:

```
aws.create_policy(
    _policy_name(bucket_name, readwrite=True),
    get_policy_document(bucket_name, True),
)
aws.create_policy(
    _policy_name(bucket_name, readwrite=False),
    get_policy_document(bucket_name, False),
)
```

which is valid from a PEP8-perspective and it's visually clear what's an argument of the function, etc...BUT, you two are the python ninjas.

Reply:
It is acceptable to PEP8, but I think PEP8 is wrong on this one.

Reply:
Cool.  I would rather follow a strict set of rules than having to debate _personal opinion and preference_.

However I happen to agree on this preference which is also valid pep8 @xoen @andyhd so I'll try and hack my formatter to use hanging!

Reply:
![What if PEP8 was wrong](https://i.imgflip.com/1wbmdv.jpg)